routes/api.php

```diff
@@ -250,6 +250,12 @@ function () {
                         Route::post('/navaids', 'Admin\\NavaidAdminController@createNavaid');
 
                         Route::get('/stand-types', 'Admin\\StandAdminController@getTypes');
+
+                        Route::get('/controller-positions', 'Admin\\AllPositionsController');
+                        Route::get('/notifications', 'Admin\\NotificationAdminController@getNotifications');
+                        Route::post('/notifications', 'Admin\\NotificationAdminController@createNotification');
```

Comment:
Could do a route group here with a prefix, keep things tidy?

Reply:
sorted this too